mlflow/entities/source_type.py

```diff
@@ -1,10 +1,11 @@
 class SourceType(object):
     """Enum for originating source of a :py:class:`mlflow.entities.Run`."""
-    NOTEBOOK, JOB, PROJECT, LOCAL, UNKNOWN = range(1, 6)
+    NOTEBOOK, JOB, CLUSTER, PROJECT, LOCAL, UNKNOWN = range(1, 6)
 
     _STRING_TO_SOURCETYPE = {
         "NOTEBOOK": NOTEBOOK,
         "JOB": JOB,
+        "CLUSTER": CLUSTER,
```

Comment:
I don't think we need to add `CLUSTER` as a source type, it'll show up separately from the "source" that generated the run in the MLflow UI (i.e. a notebook/job/local file will still be the run's source). Thus I'd undo changes to this file.